client/app/components/Field.jsx

```diff
@@ -93,17 +93,21 @@ export const PasswordField = translate()(
 )
 
 export const DropdownField = translate()((props) => {
-  const { value, options } = props
+  const { value, options, onChange, onInput } = props
 
   return (
     <FieldWrapper {...props}>
-      <select className='account-field-dropdown'>
-        {options.map(optionValue => {
-          return <option
+      <select
+        className='account-field-dropdown'
+        value={value}
```

Comment:
This is a React thing. See https://facebook.github.io/react/docs/forms.html#the-select-tag